packages/api/cms-api/src/mikro-orm/mikro-orm.module.ts

```diff
@@ -69,6 +70,7 @@ export function createOrmConfig({ migrations, ...defaults }: MikroOrmNestjsOptio
                 { name: "Migration20230302145445", class: Migration20230302145445 },
                 { name: "Migration20220905145606", class: Migration20220905145606 },
                 { name: "Migration20230209111818", class: Migration20230209111818 },
+                { name: "Migration20230808085034", class: Migration20230808085034 },
```

Comment:
Nit: Please move this migration so that the migrations are sorted by timestamp (= same as the imports)